pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -0,0 +1,2436 @@
+      kubernetes-repo: census-rm-kubernetes-optional-repo}
+
+- name: "CI Deploy Regional Counts"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure,
+                  ci-acceptance-tests]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Deploy Infrastructure"]
+  - get: census-rm-kubernetes-optional-repo
+    trigger: true
+  - get: toolbox-master
+    trigger: true
+    passed: ["Build Toolbox Latest"]
+  - get: toolbox-docker-image-gcr
+    params:
+      skip_download: true
+  - get: census-rm-deploy
+  - task: apply-deployment
+    file: census-rm-deploy/tasks/kubectl-apply-deployment.yml
+    on_failure: *slack_failure_alert_ci
+    params:
+      SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+      GCP_PROJECT_NAME: ((ci-gcp-project-name))
+      KUBERNETES_CLUSTER: ((ci-kubernetes-cluster-name))
+      KUBERNETES_DEPLOYMENT_NAME: regionalcounts
+      KUBERNETES_SELECTOR: app=regionalcounts
+      KUBERNETES_FILE_PATH: kubernetes-repo/optional
+      KUBERNETES_FILE_PREFIX: regional-counts-dev
+      WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
+    input_mapping: {
+      docker-image-resource: toolbox-docker-image-gcr,
+      kubernetes-repo: census-rm-kubernetes-optional-repo}
+
+
+
+- name: "CI Acceptance Tests"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure,
+                  ci-acceptance-tests,
+                  docker-builds]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Deploy Action-Scheduler", 
+             "CI Deploy Case-API", 
+             "CI Deploy Case-Processor", 
+             "CI Deploy UAC QID Service", 
+             "CI Deploy PubSub Service",
+             "CI Deploy Print File Service",
+             "CI Deploy Fieldwork Adapter",
+             "CI Deploy Notify Processor",
+             "CI Deploy Notify Stub",
+             "CI Deploy Exception Manager",
+             "CI Deploy Toolbox",
+             "CI Deploy Event Latency Monitor",
+             "CI Deploy Rabbit Monitor",
+             "CI Deploy Regional Counts"]
+  - get: acceptance-tests-repo
+  - get: acceptance-tests-docker-image-gcr
+    trigger: true
+    params:
+      skip_download: true
+  - get: census-rm-kubernetes-microservices-repo
+    trigger: true
+    passed: ["CI Deploy Action-Scheduler", 
+             "CI Deploy Case-API", 
+             "CI Deploy Case-Processor", 
+             "CI Deploy UAC QID Service", 
+             "CI Deploy PubSub Service",
+             "CI Deploy Print File Service",
+             "CI Deploy Fieldwork Adapter",
+             "CI Deploy Notify Processor",
+             "CI Deploy Notify Stub",
+             "CI Deploy Exception Manager"]
+  - get: census-rm-kubernetes-optional-repo
+    trigger: true
+    passed: ["CI Deploy Toolbox",
+             "CI Deploy Event Latency Monitor",
+             "CI Deploy Rabbit Monitor",
+             "CI Deploy Regional Counts"]
+  - get: action-scheduler-master
```

Comment:
Acceptance tests are triggered by the github repo resources now, making it much easier to trace which actual commit triggered/changed in any test run.